media-gfx/iscan/iscan-2.30.4.2.ebuild

```diff
@@ -0,0 +1,140 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_DOC="userg_revQ"
+inherit autotools desktop flag-o-matic
+
+# HINTS:
+# -> non-free modules are x86 and amd64 only
+# -> iscan frontend needs non-free modules
+# -> sane-epkowa should be usable on every arch
+# -> ${P}-${SRC_REV}.tar.gz    (for gcc 3.2/3.3)
+# -> ${P}-${SRC_REV}.c2.tar.gz (for gcc 3.4 or later)
+
+# FIXME:
+# iscan doesn't compile w/o libusb, this should be fixed somehow.
+
+# TODO:
+# (re)add closed-source binary modules which are needed for some scanners.
+
+DESCRIPTION="EPSON Image Scan! for Linux (including sane-epkowa backend)"
+HOMEPAGE="http://download.ebz.epson.net/dsc/search/01/search/?OSC=LX"
+SRC_URI="http://support.epson.net/linux/src/scanner/iscan/${PN}_$(ver_rs 3 -).tar.gz
+	doc? (
+		https://dev.gentoo.org/~flameeyes/avasys/${MY_DOC}_e.pdf
```

Comment:
I know you can't do much about this, and this is still hosted on GENTOO_MIRRORS, but I guess at some point I will open a bug about replacing this dead URL. I'll find a place to host these files before that. Just FYI, no action needed now.

Reply:
Actually I couldn't build this with `USE="doc"` due to these files being unfetchable. I think you need to remove them, or send them to me via e-mail and I'll upload them somewhere.

Reply:
Oops.  I read that part but forgot about it.  I thought developer files were always supposed to be available even if the developer retired.  Although when that was explained to me years ago, I thought that was a dubious remark at the time.   As it turns out the filenames have also changed.  They can be downloaded here: http://download.ebz.epson.net/dsc/search/01/search/?OSC=LX 

1. Type V500 under product name and click the search button.
2. Click download button next to Linux All-in-one package.
3. Click Accept
4. Click Download for each doc file iscan_man_e.pdf and iscan_man_j.pdf

I can also email them to you but I don't know what your email address is. 
I have already changed the doc filename in the ebuild but I will need a new location for where to download the files so I can push all that up at once.

Reply:
Also, is there a way to avoid this message (i.e. stopping default behavior attempting to unpack a file that can't be unpacked)?

"unpack iscan_man_e.pdf: file format not recognized. Ignoring."

Reply:
Actually disregard that, my testing setup didnt have GENTOO_MIRRORS set so it couldnt fetch those from the original source. I'll do another test runs and merge if no issues arise.